packages/cms-kit/src/atoms/baseInput.tsx

```diff
@@ -7,7 +7,13 @@ import {cssRule} from '@karma.run/react'
 export enum InputType {
   Text = 'text',
   Checkbox = 'checkbox',
-  Radio = 'radio'
+  Radio = 'radio',
+  Email = 'email',
+  Password = 'password',
+  Number = 'number',
+  Search = 'search',
+  Tel = 'tel',
+  Url = 'url'
```

Comment:
@per2plex There are more `type` that we have to support. On output it should be possible to choose witch type to use.